Compatibility/Macmini6,1.json

```
@@ -0,0 +1,19 @@
+{
+	"MacName": "Mac mini (Late 2012)",
+	"Author": "@ajsap",
+	"Details": "Works perfectly fine with KEXT patch, except FileVault is not available.",
+	"Works": [
+	"Metal Graphics",
+	"Sound",
+	"I/O Ports",
+  "AirDrop"
+    ],
+    "Unknown": [
+	"One thing you are unsure about"
+    ],
```

```suggestion
      "Approved": ["@BenSova"]
	"Details": "Works perfectly fine with KEXT patch, except FileVault is not available.",
	"Works": [
	    "Graphics Acceleration",
	    "Sound",
	    "I/O Ports",
          "AirDrop"
    ],
    "Unknown": [],
```
Just a couple of changes before I merge this